examples/single-org-per-testfile/myTest.nut.ts

```diff
@@ -0,0 +1,65 @@
+// this file is an example of doing everything "manually".  Compare it to the myTestWithSession to see how it's changed
+// this file also presumes that somewhere else, a global default devhub has already been auth'd
+// used to verify example code.  In real life, use
+// import { execCmdAsync, stubs } from 'cli-plugins-testkit';
+import { execCmdAsync } from '../../src/execCmd';
+import TestSession, { stubs } from '../../src/testSession';
+
+import { fs, fs as fsCore } from '@salesforce/core';
+import { expect } from 'chai';
+
+// should be unique across tests
+const testAlias = 'myTestOrg';
+const projectDir = 'testProjectDataCommands';
+
+describe('test data commands', () => {
+  before(async () => {
+    if (process.env.LEFTOVERS_MODE && fs.existsSync(projectDir)) {
+      // check to see if project already exists.  If so, exit so it can be reused.  If not, we still need to create it
+      return;
+    }
+    // optional stub ~/.sfdx directory.
+    // This test creates and deletes an org by alias, but this might be useful to protect existing configuration when working with plugins that modify it (ex: org:list --clean)
+    stubs.stubHomeDir(testAlias);
+    // stub cwd to be that project.  prevents you from having to pass in cwd to exec on every command
+    stubs.stubCWD(projectDir);
```

Comment:
I see these aren't implemented yet. I would be curious to see how you envision this working so you don't have to pass them into exec. Also for the stubHomeDir too.